mmv1/templates/terraform/examples/bigquery_job_copy.tf.tmpl

```diff
@@ -68,7 +69,7 @@ EOF
     kms_key_name = google_kms_crypto_key.crypto_key.id
```

Comment:
Where is this crypto key resource defined?

It doesn't get created with every test run, does it?

Reply:
Yeah, they are. These tests aren't the ones that are failing and I'm not addressing that in this change.